review/fileToReview.js

```diff
@@ -1,5 +1,5 @@
 
-var domain = "bank.local.fr"
```

Comment:
This variable is not modified so prefer const

Reply:
oui, voire utiliser une variable d'environnement.